packages/cli/index.js

```diff
@@ -143,6 +144,11 @@ commander
       stdio: 'inherit'
     })
 
+    await execa('pod', ['install'], {
```

Comment:
this should only execute on mac, since there is no way to do iOS development on windows or linux